src/app.ts

```diff
@@ -1,10 +1,11 @@
-import { validateCli, read } from '@1password/op-js';
+import { validateCli, read, setGlobalFlags } from '@1password/op-js';
 import * as cache from './cache';
 import path from 'path';
 import fs from 'fs';
 
 type PluginConfig = {
   cliPath?: string;
+  flags?: {};
```

Comment:
Can you change that type to e.g. `Record<string, any>`? Using `{}` as a type means that this is an object without any keys assigned to it.